RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -363,104 +363,156 @@ boolean LX_lockPicking()
 	return get_property("lockPicked").to_boolean();
 }
 
-boolean LX_phatLootToken()
+float estimateDailyDungeonAdvNeeded()

```

Comment:
:+1:
There is some way to calculate exactly which encounters you'll receive and the order you'll receive them in since the dungeon is procedurally generated every day (http://noblesse-oblige.org/calendar/ lists the monster encounters every day, I have never seen it be wrong) but this is an excellent alternative.
It has annoyed me when I see my test runs get the cubeling drops late on day 1 in an LKS run and then it doesn't bother doing the dungeon because it has less than 15 adventures constantly (but then I remember LKS runs take 4 days in Normal anyway and I have the reward permed so it doesn't matter).

Reply:
oh nice. I was not aware of this calculator.
thank you.